react/graphql/insertCoupon.graphql

```diff
@@ -1,5 +1,33 @@
 mutation insertCoupon($text: String) {
   insertCoupon(text: $text) {
-    code
+    items {
+      additionalInfo {
+        brandName
+      }
+      detailUrl
+      id
+      imageUrl
+      listPrice
+      measurementUnit
+      name
+      price
+      productId
+      quantity
+      sellingPrice
+      skuName
+      skuSpecifications {
+        fieldName
+        fieldValues
+      }
+    }
+    marketingData {
```

Comment:
the query needed for this component is only `marketingData` and `messages` the rest you won't need to use.